tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -132,40 +146,96 @@ def load_test_data(self, sql_statements=None):
                 self.assertIsNotNone(response.result())
                 self.assertIsNone(response.exception())
 
-    class DropRowsTestBase(BigQueryTestBase):
+        def assertTableDoesNotExist(self, fq_table_name):
+            """
+            Helper function to assert a table doesn't exist.
+
+            Asserts trying to get a row count from a table that does not
+            exists raises an exception.
+
+            :param fq_table_name: a fully qualified table name
+            """
+            response = self.client.query(
+                f"select count(*) from `{fq_table_name}`")
+            self.assertRaises(gc_exc.GoogleCloudError,
+                              response.result,
+                              timeout=15)
+
+        def assertSandboxTableContentsMatch(self, fq_table_name, field, values,
+                                            message):
+            """
+            Helper function to assert a  sandbox tables contents.
+
+            :param fq_table_name: table to query
+            :param field: the field to retrieve data for
+            :param values: a list of values to expect back from the query
+            :param message: a failure message for this type of assertion
+            """
+            query = f"select {field} from `{fq_table_name}`"
+            response = self.client.query(query)
+            # start the job and wait for it to complete
+            result_list = list(response.result())
+
+            result_id_list = self.create_values_list(result_list, 0)
+
+            self.assertCountEqual(values, result_id_list, message)
+
+        def assertOMOPTableContentsMatch(self, fq_table_name, values, message):
+            """
+            Helper function to assert a tables contents.
+
+            :param fq_table_name: table to query
+            :param values: a list of values to expect back from the query
+            :param message: a failure message for this type of assertion
+            """
+            table_name = fq_table_name.split('.')[-1]
+            query = f"select {table_name}_id from `{fq_table_name}`"
+            response = self.client.query(query)
+            # start the job and wait for it to complete
+            result_list = list(response.result())
+
+            result_id_list = self.create_values_list(result_list, 0)
+
+            self.assertCountEqual(values, result_id_list, message)
+
+        @staticmethod
```

Comment:
Food for thought: I was curious about the appropriate usage of `@staticmethod`. Interestingly Google's py style guide actually says to avoid this decorator entirely: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#2174-decision

It's probably fine in this case, but perhaps we should establish some codebase conventions.